sanity-suite/src/testBookSuites/sourceConfig.js

```diff
@@ -0,0 +1,41 @@
+import sourceConfig1ExpectedData from '../../__mocks__/sourceConfig1.json';
+import { getConfigUrl, getJSONTrimmed, getUserProvidedConfigUrl } from '../../../src/utils/utils';
+
+const sourceConfigAPISuite = {
+  id: 'sourceConfig',
+  name: 'sourceConfig Endpoint',
+  description: 'sourceConfig Endpoint',
+  testCases: [
+    {
+      id: 'sourceConfig1',
+      description: 'Call sourceConfig Endpoint',
+      inputData: [],
+      expectedResult: sourceConfig1ExpectedData,
+      triggerHandler: [
+        (resultCallback) => {
```

Comment:
`resultCallback` is not imported, how is it working?

Reply:
this is not a imported value, it is just the name of the first argument that will be passed into this callback when used by testbook code